packages/driver/cypress/e2e/commands/assertions.cy.js

```diff
@@ -368,7 +368,7 @@ describe('src/cy/commands/assertions', () => {
 
         cy.on('log:added', (attrs, log) => {
           if (log.get('name') === 'assert') {
-            logs.push(log)
+            logs?.push(log)
```

Comment:
I was seeing a ton of errors for cannot pushed to defined throughout our tests.

Reply:
I'm curious. How is `this.logs` undefined when it's defined above?

Reply:
I seemed to happen it was when I'd change a file or the spec file and Cypress would start to rerun the spec. 🤷🏻‍♀️Maybe the previous cy.on listener is still attached when the page is loading?? Seems a bit odd to me, but was tired of getting `uncaught exception` logs when debugging 😬

Reply:
should we drop a comment there to explain this behavior incase someone runs into it again and thinks the optional chainer isn't necessary?